usaspending_api/api_docs/api_documentation/spending_explorer.md

```diff
@@ -0,0 +1,95 @@
+## [Spending Explorer](#spending-explorer)
+
+**Route:** `/api/v2/spending/`
+
+**Method:** `POST`
+
+This route sends a request to the backend to retrieve spending data information through various types and filters.
+
+### Request
+
+```
+{
+    "type": "agency",
+    "filters": {
+        "fy": "2017"
+    }
+}
+```
+**Query Parameters Description**
+
+**type** - `required` - a string that contains the type of spending to explore on.
+  `explorer types` - *budget_function*, *budget_subfunction*, *federal_account*, *program_activity*, *object_class*, *recipients*, *awards*, *agency*
+
+**filters** - `optional` - integer or string index values to filter explorers on.
+  `filter options` - *budget_function*, *budget_subfunction*, *federal_account*, *program_activity*, *object_class*, *recipient*, *award*, *agency*, *fy*
```

Comment:
The filter option should actually be a string (even if the API supports int) for consistency between how we use it and what it could be used for.

Reply:
Also this is missing the parameters for selecting quarters. You should also explain that for quarters, it is cumulative and that quarters are only available 45 days after their close, and also that data is not available before FY 2017 Q2.

Reply:
Filters are documented as string only.
Quarters is added as a filter option.
Quarter logic is added to the filters definition.